kernel/arch/dreamcast/include/dc/pvr/pvr_regs.h

```diff
@@ -175,7 +175,7 @@ __BEGIN_DECLS
 #define PVR_RAM_BASE        0xa5000000  /**< \brief VRAM 32-bit, P2 area, PVR->VRAM */
 #define PVR_RAM_INT_BASE    0xa4000000  /**< \brief VRAM 64-bit, P2 area, PVR->VRAM */
 
-#define PVR_RAM_SIZE_MB     (KOS_PLATFORM_IS_NAOMI ? 16 : 8)    /**< \brief RAM size in MiB */
+#define PVR_RAM_SIZE_MB     (hardware_sys_mode(NULL) ? 16 : 8)  /**< \brief RAM size in MiB */
```

Comment:
Slight preference for `hardware_sys_mode(NULL) == HW_TYPE_RETAIL ? 8 : 16`
Otherwise you expect `HW_TYPE_RETAIL` to always be 0, which might change (even accidentally) in the future.